src/load/load/load.py

```diff
@@ -19,7 +19,7 @@ def load_secrets(project, vault_host, vault_port, auth, secrets_file, kvv2_mount
   vault_client = Vault(
     url_host_port(vault_host, vault_port),
     auth_method=auth,
-    role=project,
```

Comment:
works only for Kubernetes auth. for IAM auth, let `role` be None (argparse option added above) so hvac lib defaults it to IAM username.